.trunk/trunk.yaml

```diff
@@ -0,0 +1,43 @@
+version: 0.1
+cli:
+  version: 0.1.31-alpha
+  sha256:
+    darwin_x86_64: 15f297b0657b615609adfd1a30b99a6210a6ff7c52dc8bdfeb0053a8aff7a9bc
+    linux_x86_64: 5ed59e39cd2f0f72008834073cec429689c88e480abbe89cc36340b75c648a9f
+lint:
+  enabled:
+    - actionlint@1.6.4
```

Comment:
All our linters are installed and cached on a per-version basis by our `make` system. We don't want to break this just for trunk.

Reply:
The intention of trunk is to offload responsibility for managing/maintaining these tools. Feels like the wrong fit here.